README.md

```diff
@@ -1,138 +1,70 @@
+
+# Automatic Leveling
+
+<img align="right" height="250" src="Assets/leveling.png">
+
+AutoDuty has a leveling feature that will continuously run you through the "correct" dungeon to level through. The dungeon is typically determined by path quality, support for boss fights, and obviously the level of the dungeon.
+
+With AutoDuty, you can level a job from 15 to the max level very quickly and fully AFK, provided you have the correct gear and the option in the AD configuration to automatically the best gear.
+
+This plugin can also level trusts for you in the same way it levels your jobs. It can intelligently select trust members and level them all to max, fully AFK.
+
+# Automations
+
+AutoDuty supports a wide range of automations and integrations with other plugins. Below are just some of what AD can offer while automating instanced content:
+
+| Before runs | During the run | Between runs | After the runs |
+| -------- | -------- | -------- | -------- |
+| Move to an inn, house, or FC house <br /> Auto self- or NPC- repair <br /> Execute commands (i.e. SND scripts) <br /> Auto consume items | Loot treasure coffers <br /> Manage plugin states | Auto extract materia <br /> Auto desynth loot <br /> Auto turn in loot to GC <br /> Use AutoRetainer <br /> Auto equip best gear | Stop looping at a level <br /> Stop looping when you run out of rested XP <br /> Turn on AutoRetainer multi mode <br /> Shut down your computer |
+
+<hr />
+
+# Installation
+
+Add `https://puni.sh/api/repository/herc` to your plugin repositories and then search for `AutoDuty` in the Plugin Installer to install AutoDuty.
+
+Settings can be accessed via the Plugin Installer or using the chat command `/ad`.
+
+## Required Plugins
+
+Additionally, the following plugins are required. Right-click the name of the plugin and copy the URL to use it to add to your repositories in-game.
+
+- [vnavmesh](https://puni.sh/api/repository/veyn): Automatic navigation with waypoints
+- [Rotation Solver Reborn](https://raw.githubusercontent.com/FFXIV-CombatReborn/CombatRebornRepo/main/pluginmaster.json): Automatic rotation execution, all jobs supported
+- [Veyn's Boss Mod](https://puni.sh/api/repository/veyn) or [BossmodReborn](https://raw.githubusercontent.com/FFXIV-CombatReborn/CombatRebornRepo/main/pluginmaster.json): Automatic fight execution for bosses
+
+## Optional Plugins
+
+The following plugins are optional. They integrate with AutoDuty well and in some cases can be triggered by AD itself.
+
+- [Deliveroo](https://plugins.carvel.li): Automatically turns in items to your Grand Company
+- [AutoRetainer](https://love.puni.sh/ment.json): Automatically manages your retainers and submersibles as well as sends them out on ventures
+
```

Comment:
can you add gearsetter here

Reply:
Resolved in [0a6ef22](https://github.com/ffxivcode/AutoDuty/pull/317/commits/0a6ef227ab04ddd4d6c742645894dd80caaa7406)